runtime-config-linux.md

```diff
@@ -503,3 +503,14 @@ Its value is either slave, private, or shared.
 ```json
     "rootfsPropagation": "slave",
 ```
+
+## No new privileges
+
+Setting `noNewPrivileges` to true prevents the processes in the container from gaining additional privileges.
```

Comment:
On Wed, Jan 20, 2016 at 02:15:18PM -0800, Vish Kannan wrote:

> > +Setting `noNewPrivileges` to true prevents the processes in the
> > container from gaining additional privileges.
> 
> Why not?

Search for ‘opt-’ in the PR discussion to get previous opt-in
vs. opt-out thoughts ;).